packages/derive/src/callable_point.rs

```diff
@@ -7,11 +7,11 @@ pub fn make_callable_point(function: syn::ItemFn) -> TokenStream {
     let function_name_ident = &function.sig.ident;
     let mod_name_ident = format_ident!("__wasm_export_{}", function_name_ident);
     // The first argument is `deps`, the rest is region pointers
-    if function.sig.inputs.is_empty() {
+    if function.sig.inputs.len() < 2 {
```

Comment:
I'm not sure if I asked this question when we discussed this function, so just to confirm, do we always need the deps and envs at the beginning of the function?

Reply:
Yes,
- about `deps`: It is needed now. (technically, it is able to detect which it is defined as arg and deal with it if we modify the `callable_point` macro. This is the same as `entry_point`)
- about `env`: It is always needed. because it is passed by wasmvm.

Reply:
If so, `!=2`  is better than `< 2` ?

Reply:
the 3rd and later args are normal args of the callee point. So, it cannot be `!= 2`

Reply:
Ok, I could understand more better, thx.